masschallenge/impact-api
# [AC-4598] New build environment

Work done:
- Complete revision of the Makefile and associated targets.
- All regular developer workflows up to release/deploy should be supported by make targets.

To test:
- Checkout AC-4598 branch for impact-api and django-accelerator repos in sibling directories
- Run `make help` to understand what the targets are and should do
- Run `make build` before you try any of the other targets.
- Run `make test` to make sure everything is working.
- Review all the other targets.
